test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -3109,12 +3109,48 @@ def setup_rnn_tests():
 
 setup_rnn_tests()
 
+# Unary floating ufunc tests
+# Tests that integral arguments are properly cast to float
+unary_floating_ufuncs = (
+    'sin', 'cos', 'tan',
+    'asin', 'acos', 'atan',
+    'sinh', 'cosh', 'tanh',
+    'ceil', 'floor',
+    'exp', 'log', 'log1p', 'log2',
```

Comment:
log1p and log2 are (correctly) in this list, but their ONNX translations haven't been changed. Were they already working correctly?

Reply:
I believe so. Since they call other unary floating ufunc which have been changed, so IMO: they don't need the modifications. Please let me know if you think they should be changed. The tests did pass successfully for all the ops.